packages/components-props-analyzer/src/ComponentsList.ts

```diff
@@ -81,6 +81,7 @@ const components: Component[] = [
     {
         name: 'ModalWizard',
         packageName: '@coveord/plasma-react',
+        suffix: 'Legacy',
```

Comment:
Don't forget to adjust the import in the legacy component demo page. Otherwise it will display the props of the new mantine component in its props table.

Reply:
Thank you for pointing it out! I did not there is also legacy file that needs to be updated. Now I know!